src/components/Map/Map.jsx

```
@@ -90,7 +91,11 @@ const Map = ({
   onWaypointsBulkDelete,
   selectedPolygonId,
   editingPolygon = null,
-  drawMode = false
+  drawMode = false,
+  showDIDTooltip: externalShowDIDTooltip,
+  onShowDIDTooltipChange,
+  didTooltipAutoFade: externalDidTooltipAutoFade,
+  onDidTooltipAutoFadeChange
```

ESLint

[error] 'onDidTooltipAutoFadeChange' is defined but never used. Allowed unused args must match /^_/u.

(no-unused-vars)